### PR TITLE
patch fixing a serialization issue with rx,ry and rz gates.

### PR DESCRIPTION
**Pull request by mganahl**
The current serialization of parametrized circuits misses the case where a sympy object is a numeric constant, as for exampole sympy.pi. This problem arises for parametrized rx, ry and rz gates

This patch is an attempt at fixing this.

### CODE REVIEW

File: cirq/json_resolver_cache.py [hunk 1]
```diff
@@ -153,5 +153,8 @@ def two_qubit_matrix_gate(matrix):
         'sympy.Float': lambda approx: sympy.Float(approx),
         'sympy.Integer': sympy.Integer,
         'sympy.Rational': sympy.Rational,
+        'sympy.pi': lambda: sympy.pi,
```
Comment (dstrain115) [inline cirq/json_resolver_cache.py:156]: Could we add a test (or modify one of the existing tests) to use one of these symbols?

Thanks for contributing!

Comment (mganahl) [reply cirq/json_resolver_cache.py:156]: NP, will add it right away!
Looping in @jacksonwb, this PR should fix the serialization issues with parametrized circuits

Comment (mganahl) [reply cirq/json_resolver_cache.py:156]: @dstrain115 added some test, let me know what you think!

**Comment by mganahl** [comment]
@dstrain115 ping for review, thanks!

**Comment by mganahl** [comment]
@vtomole fixed the test!

File: cirq/json_resolver_cache.py (hunk 1 above)
Comment (vtomole) [reply cirq/json_resolver_cache.py:159]: This comment has been addressed.

**Review by vtomole — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**